src/components/swap/SwapHeader.tsx

```diff
@@ -13,14 +22,136 @@ const StyledSwapHeader = styled.div`
   color: ${({ theme }) => theme.textSecondary};
 `
 
+const Dot = styled.span`
+  height: 8px;
+  width: 8px;
+  background-color: ${({ theme }) => theme.accentActive};
+  border-radius: 50%;
+  margin: 0px 4px;
+  display: inline-block;
+`
+
+const TextHeader = styled.div<{ color: string; marginLeft?: string; isClickable?: boolean; disabled?: boolean }>`
```

Comment:
use `button` here if this component is clickable

and then you can use just use the one `disabled` prop and derive hover states based on that

```
cursor: pointer 

&:disabled {
   cursor: unset
}
```

Reply:
Hmm so I don't want this component to always be clickable / be a button because the 'Swap' header is also styled by this component, and it is not clickable.

Reply:
sounds good done, separated BuyFiatButton into a separate component and used ButtonText component